vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -0,0 +1,175 @@
+---
+sidebarDepth: 2
+---
+# Manage NGINX
+
+## Verify the NGINX Ingress Install
+We recommend setting up a test application to verify that the ingress is working.
+
+1. Create a simple application from the `Cloud Shell`:
```

Comment:
Cloud Shell is only for GKE. Let's just remove "from the Cloud Shell" in this sentence.